src/miral/static_display_config.cpp

```diff
@@ -644,20 +644,29 @@ void miral::ReloadingYamlFileDisplayConfig::auto_reload()
         if (auto const ml = the_main_loop())
         {
             ml->register_fd_handler({icf.value()}, this, [icf=icf.value(), this] (int)
+            {
+                static size_t const sizeof_inotify_event = sizeof(inotify_event);
+
+                // A union ensures buffer is aligned for inotify_event
```

Comment:
This could instead be
```
alignas(inotify_event) char buffer[sizeof(inotify_event) + NAME_MAX + 1];
```
right?

Reply:
Yes, done

Reply:
Hm, done to *one* of the copies of this logic, but not to the one directly below here?